build.assets/genproto.sh

```diff
@@ -26,7 +27,7 @@ main() {
   cp -r github.com/gravitational/teleport/* .
 
   # Generate prehog protos.
-  cd lib/prehog && buf generate
+  buf generate --template=lib/prehog/buf.gen.yaml lib/prehog/proto
```

Comment:
Could we fit prehog in one of the categories above? As in, gogo or plain protoc-gen-go?

Reply:
> Could we fit prehog in one of the categories above? As in, gogo or plain protoc-gen-go?

I'm going to move it to the protoc-gen-go section since it does use protoc-gen-go, but it also uses connect-go.

Reply:
Right. I'm not stoked to add a 3rd proto compiler, but I won't meddle with this one either.

I think we might need a separate config - IIUC, protoc-gen-connect-go replaces protoc-gen-grpc-go.